frontend/src/config/rbd/layers.json

```diff
@@ -6404,5 +6404,141 @@
+  "fs_hotspot_2024": {
+    "title": "Food Security Hotspot June 2024",
+    "type": "admin_level_data",
+    "path": "data/rbd/fsn-hotspot/fs-hotspot-1-4-2024.json",
+    "admin_level": 2,
+    "data_field": "FS_Hotspot_Jun_2024",
+    "admin_code": "admin2Pcod",
+    "opacity": 0.9,
+    "legend": [
+      {
+        "label": "Priority 1",
+        "value": 2,
+        "color": "#b30101"
+      },
+      {
+        "label": "Priority 2",
+        "value": 3,
+        "color": "#e24a32"
+      },
+      {
+        "label": "Priority 3",
+        "value": 4,
+        "color": "#fcca8b"
+      },
+      {
+        "label": "Priority 4",
+        "value": 5,
+        "color": "#fff0d9"
+      },
+      {
+        "label": "Not analyzed",
+        "value": 9,
+        "color": "#c9d2cd"
+      }
+    ],
+    "legend_text": "Joint Food Security & Nutrition Hotspot Analysis conducted by WFP, UNICEF, and partners based on SMART surveys and Cadre Harmonisé (CH) & Integrated Food Security Phase Classification (IPC) analyses"
+  },
+  "fs_hotspot_1a_2024": {
+    "title": "Food Security Hotspot June 2024",
+    "type": "admin_level_data",
+    "path": "data/rbd/fsn-hotspot/fs-hotspot-1a-2024.json",
+    "admin_level": 2,
+    "data_field": "FS_Hotspot_Jun_2024",
+    "admin_code": "admin2Pcod",
+    "fill_pattern": "right",
+    "opacity": 0.9,
+    "legend": [
+      {
+        "label": "Priority 1a",
+        "value": 1,
+        "color": "#b30101"
+      }
+    ],
+    "legend_text": " "
+  },
+  "nutr_hotspot_2024": {
+    "title": "Nutrition hotspot analysis",
+    "type": "admin_level_data",
+    "path": "data/rbd/fsn-hotspot/fsn-hotspot-2024.json",
+    "admin_level": 2,
+    "data_field": "NUT_Hotspot_Jun_2024",
+    "admin_code": "admin2Pcod",
+    "opacity": 0.9,
+    "legend": [
+      {
+        "label": "Low",
+        "value": 1,
+        "color": "#70a800"
+      },
+      {
+        "label": "Medium",
+        "value": 2,
+        "color": "#ffcc33"
+      },
+      {
+        "label": "High",
+        "value": 3,
+        "color": "#e67800"
+      },
+      {
+        "label": "Very high",
+        "value": 4,
+        "color": "#e60000"
+      },
+      {
+        "label": "Not analyzed",
+        "value": 9,
+        "color": "#c9d2cd"
+      }
+    ],
+    "legend_text": "Joint Food Security & Nutrition Hotspot Analysis conducted by WFP, UNICEF, and partners based on SMART surveys and Cadre Harmonisé (CH) & Integrated Food Security Phase Classification (IPC) analyses"
+  },
+  "integrated_hotspot_2024": {
+    "title": "Integrated food security and nutrition hotspot analysis",
+    "type": "admin_level_data",
+    "path": "data/rbd/fsn-hotspot/fsn-hotspot-2024.json",
+    "admin_level": 2,
+    "data_field": "Integrated_FS_NUT_Hotspot_Jun_2024",
+    "admin_code": "admin2Pcod",
+    "opacity": 0.9,
+    "legend": [
+      {
+        "label": "Very low",
+        "value": 1,
+        "color": "#f1f0f6",
+        "fillPattern": "right"
```

Comment:
I added a few examples here but adjust for your need before merging

<img width="1182" alt="Screenshot 2024-06-26 at 3 32 56 PM" src="https://github.com/WFP-VAM/prism-app/assets/16843267/71b5dc8c-5095-48fd-bb43-6a2b47e82918">

Fixing #1282